### PR TITLE
Fixed a few usability issues

### PR DESCRIPTION
**Pull request by datamattsson**
Our examples on SCOD uses "hpe-backend" as the default `Secret`, we should default to that with the exporter too. I found it annoying that the chart installed without issues when ignoring the "acceptEula" parameter.

This is what installation looks like now:
```
$ helm install -n hpe-storage my-hpe-array-exporter -f values-exporter-array1.yaml my-shart/hpe-array-exporter --version=1.0.0-edge --set acceptEula=false
Error: template: hpe-array-exporter/templates/hpe-array-exporter.yaml:33:16: executing "hpe-array-exporter/templates/hpe-array-exporter.yaml" at <fail "\n\nERROR: The acceptEula value must be set to true to confirm acceptance.\n\n">: error calling fail: HELM_ERR_START

ERROR: The acceptEula value must be set to true to confirm acceptance.

HELM_ERR_END
$ helm install -n hpe-storage my-hpe-array-exporter -f values-exporter-array1.yaml my-shart/hpe-array-exporter --version=1.0.0-edge --set acceptEula=no
Error: values don't meet the specifications of the schema(s) in the following chart(s):
hpe-array-exporter:
- acceptEula: Invalid type. Expected: boolean, given: string

$ helm install -n hpe-storage my-hpe-array-exporter -f values-exporter-array1.yaml my-shart/hpe-array-exporter --version=1.0.0-edge --set acceptEula=true
NAME: my-hpe-array-exporter
LAST DEPLOYED: Wed Jul 28 15:56:59 2021
NAMESPACE: hpe-storage
STATUS: deployed
REVISION: 1
NOTES:
Use of this software is governed by the HPE End User License Agreement at
https://www.hpe.com/us/en/software/licensing.html.
```

Signed-off-by: Michael Mattsson <michael.mattsson@gmail.com>

### CODE REVIEW

**Review by raunakkumar — APPROVED** [review]

File: helm/charts/hpe-array-exporter/values.yaml [hunk 1]
```diff
@@ -9,7 +9,7 @@ acceptEula: false
 # The name of a Secret in the same namespace as the Helm chart installation
 # providing storage array access information: "address" (or "backend"),
 # "username", and "password".
-arraySecret: my-array-secret
+arraySecret: hpe-backend
```
Comment (rkumpf) [inline helm/charts/hpe-array-exporter/values.yaml:12]: Should we use this in the YAML files, too?

**Review by rkumpf — APPROVED** [review]
Nice, thanks.  I didn't find the "fail" directive.

File: helm/charts/hpe-array-exporter/values.yaml (hunk 1 above)
Comment (datamattsson) [reply helm/charts/hpe-array-exporter/values.yaml:12]: I'm torn. Adding for now.

File: yaml/array-exporter/edge/hpe-array-exporter-secret.yaml [hunk 2]
```diff
@@ -15,9 +15,9 @@
...
 apiVersion: v1
 metadata:
-  name: my-array-secret
+  name: hpe-backend
   namespace: hpe-storage
 stringData:
   address: 192.168.1.1
-  username: myuser
-  password: mypassword
+  username: admin
```
Comment (rkumpf) [inline yaml/array-exporter/edge/hpe-array-exporter-secret.yaml:22]: Do we want to put credentials here?  Seems like they'll need to be edited regardless.

Comment (datamattsson) [reply yaml/array-exporter/edge/hpe-array-exporter-secret.yaml:22]: This is for consistency of the previous examples.

**Review by rkumpf — APPROVED** [review]

**Pull request merged**